game-core/src/main/java/games/strategy/util/PointFileReaderWriter.java

```diff
@@ -40,30 +50,21 @@ private PointFileReaderWriter() {}
     checkNotNull(stream);
 
     final Map<String, Point> mapping = new HashMap<>();
-    try (Reader inputStreamReader = new InputStreamReader(new CloseShieldInputStream(stream), StandardCharsets.UTF_8);
-        LineNumberReader reader = new LineNumberReader(inputStreamReader)) {
-      @Nullable
-      String current = reader.readLine();
-      while (current != null) {
-        if (current.trim().length() != 0) {
-          readSingle(current, mapping);
-        }
-        current = reader.readLine();
-      }
-    }
+    readStream(stream, current -> readSingle(current, mapping));
     return mapping;
   }
 
-  private static void readSingle(final String line, final Map<String, Point> mapping) throws IOException {
-    final StringTokenizer tokens = new StringTokenizer(line, "", false);
-    final String name = tokens.nextToken("(").trim();
-    if (mapping.containsKey(name)) {
-      throw new IOException("name found twice:" + name);
+  private static void readSingle(final String line, final Map<String, Point> mapping) {
+    final Matcher matcher = singlePointPattern.matcher(line);
+    if (matcher.find()) {
+      final String name = matcher.group(1);
```

Comment:
While we have context, if you can, are there any better variable names for this `name` variable? Name of what?